src/rulesets/asyncapi/index.json

```diff
@@ -0,0 +1,465 @@
+{
+  "formats": [
+    "asyncapi2"
+  ],
+  "rules": {
+    "asyncapi2-channel-path-no-empty-parameter-declaration": {
+      "description": "Channel path should not have empty parameter substitution pattern.",
+      "recommended": true,
+      "type": "style",
+      "given": "$.channels.*~",
+      "then": {
+        "function": "pattern",
+        "functionOptions": {
+          "notMatch": "{}"
+        }
+      }
+    },
+    "asyncapi2-channel-path-no-query-nor-fragment": {
+      "description": "Channel path should not include a query (`?`) or a fragment (`#`) delimiter.",
+      "recommended": true,
+      "type": "style",
+      "given": "$.channels.*~",
+      "then": {
+        "function": "pattern",
+        "functionOptions": {
+          "notMatch": "[\\?#]"
+        }
+      },
+      "tags": [
+        "given"
```

Comment:
@philsturgeon What are those tags for? Do we need them?

Reply:
I think tags are an artifact which was going to be used for something, but never was. @P0lip can you confirm? If so, yeah lets kill off the tags.

Reply:
I don't know what the purpose of tags is. They had been added added before I started to work on spectral.

Reply:
@philsturgeon So 💀 them with 🔥 ?

Reply:
💣💥

Reply:
They're gone, now.